### DISCUSSION TITLE
Static Adapter page w/ import .svelte file from $lib error

### DISCUSSION THREAD

**Discussion by archoda** [sveltejs/kit#3857] [Q&A]
With Sveltekit's adapter-static. When I have a page with an import to a component file, I'm getting a 500 document not found error.

Is there something else I need to be doing?

Here's my test:

/routes/test/index.svelte
```
<script>
import NavMain from '$/lib/components/nav-main.svelte';
<script>

<NavMain />
<h1>Page Test</h1>
```

Here's the terminal error
```
> Using @sveltejs/adapter-static
ReferenceError: document is not defined
...
> 500 /test
```

**Comment by mustofa-id** [comment]
is `'$/lib/components/nav-main.svelte';` really correct path? I think is `$lib` not `$/lib`
